src/Chip/Chip.js

```diff
@@ -0,0 +1,168 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import DeleteIcon from '../svg-icons/navigation/cancel';
+
+export const styleSheet = createStyleSheet('Chip', (theme) => {
+  const { palette, shadows, transitions } = theme;
+  return {
+    root: {
+      fontSize: 13,
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      height: 32,
+      outline: 'none',
```

Comment:
That was carried over from `master`, but can't imagine why it was needed. Removed.